config/confighttp/confighttp.go

```diff
@@ -280,6 +281,9 @@ type ServerConfig struct {
 	// Additional headers attached to each HTTP response sent to the client.
 	// Header values are opaque since they may be sensitive.
 	ResponseHeaders map[string]configopaque.String `mapstructure:"response_headers"`
+
+	// Compression configures the list of compression algorithms the server can accept. Default: ["", "gzip", "zstd", "zlib", "snappy", "deflate"]
```

Comment:
I'm not sure if the name is clear enough for what this option is, what do you think of something like `CompressionAlgorithms` or `SupportedCompression`?